Doc/whatsnew/3.8.rst

```diff
@@ -236,6 +236,10 @@ now return ``False`` instead of raising :exc:`ValueError` or its subclasses
 characters or bytes unrepresentable at the OS level.
 (Contributed by Serhiy Storchaka in :issue:`33721`.)
 
+:func:`~os.path.expanduser` on Windows now prefers the :envvar:`USERPROFILE`
+environment variable and does not use :envvar:`HOME`, which is not normally set
+for regular user accounts.
+
```

Comment:
I'm not sure this is in the right section; I'd expect it in "Changes in the Python API" rather than "Improved Modules".

Reply:
I mean, it clearly belongs in the section about `os.path`. I'll add it to both.